birthdays/main.py

```diff
@@ -38,8 +38,8 @@ def main():
     )
 
     bdays = bdays.sort_values("days_until_bday")
-
-    html_body = email_util.get_all_bdays_email_body(bdays)
+    
+    html_body = email_util.get_all_bdays_email_body_v2(bdays)
```

Comment:
v2 is niet handig, heeft weinig betekenis
over 3 maanden weten we het verschil tussen v1 en v2 niet meer